infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterDataV1.kt

```diff
@@ -52,6 +33,16 @@ data class FrameConverterStringV1(val value: String) {
     }
 }
 
+data class FrameConverterQueryParamsV1(
+    @JsonProperty("koordinaatisto") val coordinateSystem: Srid = Srid(3067),
+    @JsonProperty("geometriatiedot") val featureGeometry: Boolean = false,
+    @JsonProperty("perustiedot") val featureBasic: Boolean = true,
+    @JsonProperty("lisatiedot") val featureDetails: Boolean = true,
+)
```

Comment:
Tämä kuitenkin koostaa erilliset paluuarvoasetus-queryParamit dataluokaksi jota voi passailla eteenpäin kätevämmin ja sisältää niille myös oletusarvot.